src/Aspire.Dashboard/Components/Layout/MainLayout.razor.cs

```diff
@@ -78,6 +82,23 @@ protected override async Task OnInitializedAsync()
 
         var result = await JS.InvokeAsync<string>("window.getBrowserTimeZone");
         TimeProvider.SetBrowserTimeZone(result);
+
+        // ShowMessageBarAsync must come after an await. Otherwise it will NRE.
+        // I think this order allows the message bar provider to be fully initialized.
+        await MessageService.ShowMessageBarAsync(options =>
```

Comment:
Does this always show up? Where's the auth check?

Reply:
True true true